Pod/Classes/HairlineView/README.md

```diff
@@ -0,0 +1,48 @@
+# HairlineView
+
+A UIView that can be defined by axis, thickness and color
+
+## Default Appearance
+
+<details>
+<summary>Screenshots</summary>
+
+<p>
+<img src="HairlineTableCells.png" width="200">
+</p>
+
+</details>
+
+### Quick Start
+
+A simple 1 pixel, horizontal, darkGray hairline:
```

Comment:
```suggestion
A simple 1-point, horizontal, dark gray hairline:
```

Reply:
This seems to suggest that it is one pixel:
`CGFloat(1.0 / UIScreen.main.scale)`

Reply:
My mistake. Updated suggested change.